test/s3path.jl

```diff
@@ -364,28 +364,42 @@ end
     # `s3_list_versions` returns versions in the order newest to oldest
     versions = [d["VersionId"] for d in reverse!(s3_list_versions(aws, bucket_name, key_version_file))]
     @test length(versions) == 2
-    @test read(S3Path(bucket_name, key_version_file; config=aws, version=first(versions)), String) == "data.v1"
-    @test read(S3Path(bucket_name, key_version_file; config=aws, version=last(versions)), String) == "data.v2"
-    @test isequal(read(S3Path(bucket_name, key_version_file; config=aws, version=last(versions)), String),
+    v1 = first(versions)
+    v2 = last(versions)
```

Comment:
Personal preference here:
```suggestion
    v1, v2 = first(versions), last(versions)
```